services/core-web/src/components/modalContent/NoticeOfDepartureModal.tsx

```diff
@@ -423,6 +381,15 @@ const NoticeOfDepartureModal: React.FC<
             </Button>
           ) : (
             <>
+              <Popconfirm
```

Comment:
this was a UI thing that bothered me- we had it like [Update][Cancel] which is the reverse of what we usually do. There was also a class on it that made the border size change when you hovered on it.